mod/pages/upgrades/2012061800.php

```diff
@@ -1,49 +0,0 @@
-<?php
-/**
```

Comment:
yes, it has been around since 1.8.... we do not support full upgrade paths all the way from 1.8 to 3.0, so this can be removed without any issue as intermittent upgrades will have taken care of the upgrade script